be/src/olap/comparison_predicate.cpp

```diff
@@ -159,60 +159,54 @@ COMPARISON_PRED_COLUMN_BLOCK_EVALUATE(GreaterEqualPredicate, >=)
                                         .get_data();                                               \
             auto& nested_col = nullable_col->get_nested_column();                                  \
             if (nested_col.is_column_dictionary()) {                                               \
-                if constexpr (std::is_same_v<type, StringValue>) {                                 \
+                if constexpr (std::is_same_v<T, StringValue>) {                                    \
```

Comment:
Is this ```std::is_same_v<T, StringValue>``` necessary?
If a column is ColumnDict, then its TYPE in predicate can only be StringValue.

Reply:
Yes, if I remove this judgment, the compilation will report an error